story/admin.py

```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
 
+
```

Comment:
The pull request adds two blank lines to the `admin.py` file of the `story` application. While no significant logic changes are present, it is advisable to ensure that such additions contribute meaningfully to code readability or maintainability. If these changes are preparatory for future updates, consider adding comments to clarify their purpose.